shell/src/dapps/o-marketlisting/pages/MarketlistingDetail.svelte

```diff
@@ -153,9 +159,11 @@ function checkIsOpenNow(timesArray) {
   if (!timesArray) {
     return;
   }
-  var d = new Date(); // current time
+  const baliTime = dayjs().tz("Asia/Makassar");
+  var d = baliTime.$d; // current time
```

Comment:
looks like an internal property. Please check if there is a better (documented) way to get the date.